wallet/wallet-cli-lib/src/commands/helper_types.rs

```diff
@@ -42,3 +44,47 @@ impl CliUtxoTypes {
         }
     }
 }
+
+#[derive(Debug, Clone, Copy, ValueEnum)]
+pub enum CliUtxoState {
+    Confirmed,
+    Conflicted,
+    InMempool,
+    Inactive,
+    Abandoned,
+}
+
+impl Display for CliUtxoState {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            CliUtxoState::Confirmed => f.write_str("confirmed"),
```

Comment:
I know this is dumb... but please capitalize the strings since they're independent states and not part of something else, linguistically speaking.

Reply:
Alright. No problem. Please add a comment explaining this.

Reply:
commented